src/OrchardCore.Modules/OrchardCore.ContentFields/Migrations.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using OrchardCore.Data.Migration;
+using OrchardCore.ContentFields.Indexing;
+
+namespace OrchardCore.ContentFields
+{
+    public class Migrations : DataMigration
+    {
+        public int Create()
+        {
+            SchemaBuilder.CreateMapIndexTable(nameof(TextFieldIndex), table => table
+                .Column<string>("ContentType", column => column.WithLength(255))
```

Comment:
This has to match the values from the other tables.

Reply:
Changed to `.Column<string>("ContentType", column => column.WithLength(ContentItemIndex.MaxContentTypeSize))`

Reply:
Let me know if we should also use it on ContentPart and ContentField since they are also both GUID's

Reply:
What is a Guid also?

Reply:
They are unique random strings ... maybe not guid or uuid ... but something 😄

Reply:
I think we should only care about the lenght of them though 😉

Reply:
I think we should maybe eventually add the ContentPart and ContentField keys to the IndexingConstants.cs that's what I meant.

Reply:
And also add something for MaxContentPartSize, MaxContentFieldSize